react/features/base/i18n/i18next.js

```diff
@@ -46,6 +49,8 @@ const options = {
     load: 'languageOnly',
     ns: [ 'main', 'languages', 'countries' ],
     react: {
+        // re-render when a new resource bundle is added
+        bindI18nStore: 'added',
```

Comment:
I'm confused as to how is this working?

Reply:
All the labels/strings are stored separately in i18next's store. Whenever a new bundle(which contains labels/strings) is added an `added` event is emitted, which forces a re-render - remember we have acces to `t` using their HOC or hook.